travvy/src/components/Onboarding.vue

```diff
@@ -4,19 +4,27 @@
     
     <!-- Onboarding fields that the user must fill in in order to get personalized recommendations --> 
     <h1>🎉 Welcome to Travvy, we'd love to learn more about you! 🎉</h1>
+    
+  <form>
     <h3>Please fill in the following information in order to get a custom travel experience.</h3>
     <br>
+    <p v-if="errors.length">
+    <b class="red">Please correct the following error(s):</b>
+    <!-- this will bind all the errors together after the form is completed and return them to the user-->
+      <u1 class="red">
+        <li v-bind="error in errors"> {{ errors }}</li>
+      </u1>
+    </p>
     <br>
-
-    <label>Please input your email here: </label>
-    <input type="text" class="info" v-model="email">
+    <label><strong>Please verify your email here (required): </strong></label>
+    <input type="text" class="info" v-model="email" placeholder="Input Email here" required>
```

Comment:
Good thinking with adding the "strong" HTML element for required information, great indicator so it improves our user experience!